### ISSUE TITLE
[Q] Is normalization needed before running mRSC

### ISSUE THREAD

**Issue by JonathanTaws** [jehangiramjad/tslib#3]
Hi,

Thanks for making available your implementation of mRSC! Given that multidimensional timeseries can have different scales, is it needed/beneficial in theory to perform normalization before feeding in the data?

**Comment by vishalmisra** [comment]
Yes normalization always helps, it allows you to weigh different metrics appropriately.

**Comment by JonathanTaws** [comment]
@vishalmisra would there be a recommended way of doing normalization over the multivariate timeseries? Normalization can introduce biases in the data and the relationships the algorithm is trying to learn.
